core/trino-main/src/main/java/io/trino/connector/informationschema/InformationSchemaPageSource.java

```diff
@@ -280,6 +280,7 @@ private void addTablesRecords(QualifiedTablePrefix prefix)
     {
         Set<SchemaTableName> tables = listTables(session, metadata, accessControl, prefix);
         Set<SchemaTableName> views = listViews(session, metadata, accessControl, prefix);
+        // TODO define a type for materialized views
```

Comment:
could you put link to ticket for TODO?

Reply:
Created https://github.com/trinodb/trino/issues/8207